src/components/ChangelogDialog/components/Item/Item.scss

```diff
@@ -17,17 +17,18 @@ $metaWidth: 124px;
     }
 
     &__label-new {
-        margin-top: 4px;
+        margin-top: 8px;
     }
 
     &__content {
         flex: 1;
-        margin-left: 16px;
+        margin-left: 20px;
     }
 
     &__title {
         margin: 0;
-        @include mixins.text-subheader-3();
+        @include mixins.text-subheader-2();
+        font-weight: 500;
```

Comment:
mixin already has font-weight. Besides it uses variable for that, not a fixed value

Reply:
fixed, thx!